docs/getting-started-guides/juju.md

```diff
@@ -197,28 +205,41 @@ We can add node units like so:
 juju add-unit kubernetes
 ```
 
+Or multiple units at one time:
+
+```shell
+juju add-unit -n3 kubernetes
+```
 
 ## Tear down cluster
 
+We recommend that you use the `kube-down.sh` command when you are done using
+the cluster, as it properly brings down the cloud and removes some of the
+build directories.
+
 ```shell
 ./kube-down.sh
 ```
 
-or destroy your current Juju environment (using the `juju env` command):
+If you want stop the servers you can destroy your current Juju environment
+(using the `juju env` command):
 
 ```shell
 juju destroy-environment --force `juju env`
 ```
 
-
 ## More Info
 
-The Kubernetes charms and bundles can be found in the `kubernetes` project on
-github.com:
+Juju works with charms and bundles to deploy solutions. The code that stands up
+a Kubernetes cluster is done in the charm code. The charm is built from using
+a layered approach to keep the code smaller and more focused on the operations
+of Kubernetes.
 
- - [Bundle Repository](http://releases.k8s.io/{{page.githubbranch}}/cluster/juju/bundles)
-   * [Kubernetes master charm](https://releases.k8s.io/{{page.githubbranch}}/cluster/juju/charms/trusty/kubernetes-master)
-   * [Kubernetes node charm](https://releases.k8s.io/{{page.githubbranch}}/cluster/juju/charms/trusty/kubernetes)
+The Kubernetes layer and bundles can be found in the `kubernetes`
+project on github.com:
+
+ - [Bundle location](https://github.com/kubernetes/kubernetes/tree/master/cluster/juju/bundles)
+ - [Kubernetes charm layer location](https://github.com/kubernetes/kubernetes/tree/master/cluster/juju/layers/kubernetes)
```

Comment:
These urls were to a different repository so the can not be relative or use the docs repo. I believe they dinged us on URLs when they still pointed to github/google/kubernetes/...